.github/workflows/ci.yml

```
@@ -102,14 +102,59 @@ jobs:
         working-directory: test/package
 
       - name: "Test package"
-        run: npm run test:package
+        run: npm run test:package -- -t rollup
+
+  test-package-rolldown:
+    name: "Test package on rolldown v${{ matrix.rollup }} / node ${{ matrix.node }} / ${{ matrix.os }}"
```

actionlint (1.7.4)

property "rollup" is not defined in object type {node: number; os: string; rolldown: number}

(expression)